dracut/kickstart-genrules.sh

```diff
@@ -19,6 +19,12 @@ case "${kickstart%%:*}" in
             when_diskdev_appears "$ksdev" \
                 fetch-kickstart-disk \$env{DEVNAME} "$kspath"
         fi
+        # "cdrom:" also means "wait forever for kickstart" because rhbz#1168902
+        if [ "$kstype" = "cdrom" ]; then
+            # if we reset main_loop to 0 every loop, we never hit the timeout..
+            jobfile="$hookdir/initqueue/finished/wait-forever-for-ks-cdrom.sh"
+            echo "main_loop=0" > "$jobfile"
```

Comment:
I don't now how exactly the ``finished`` guards work, but where does ``main_loop`` come from and if it is a variable, shouldn't the condition be ``$main_loop=0``? Unless I'm missing some big thing here, the comment needs to be extended a bit.